front/app/containers/Admin/projects/all/new/index.tsx

```diff
@@ -1,28 +1,53 @@
 import React from 'react';
 
-import { Box } from '@citizenlab/cl2-component-library';
+import { Box, colors } from '@citizenlab/cl2-component-library';
 import { useSearchParams } from 'react-router-dom';
 
-import Header from '../_shared/Header';
+import { useIntl } from 'utils/cl-intl';
 
+import Search from './_shared/FilterBar/Filters/Search';
+import Calendar from './Calendar';
 import Folders from './Folders';
+import Header from './Header';
+import messages from './messages';
 import Ordering from './Ordering';
 import Projects from './Projects';
 import Tabs from './Tabs';
-import Timeline from './Timeline';
 
 const AdminProjectsListNew = () => {
   const [searchParams] = useSearchParams();
+  const { formatMessage } = useIntl();
   const tab = searchParams.get('tab');
 
   return (
-    <Box>
+    <Box
+      bgColor={colors.white}
+      w="100%"
+      h="100vh"
+      px="51px"
+      py="45px"
+      maxWidth="1400px"
+    >
       <Header />
-      <Tabs />
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="center"
+        borderBottom={`1px solid ${colors.grey200}`}
+        height="44px"
+      >
+        <Tabs />
+        {[null, 'timeline'].includes(tab) && (
```

Comment:
I think the tab will now be calendar so we should change this
```suggestion
        {[null, 'calendar'].includes(tab) && (
```